schema.sql

```diff
@@ -0,0 +1,27 @@
+-- Drops database if it exists currently --
+DROP DATABASE IF EXISTS heroDB;
+-- Creates new database --
+CREATE DATABASE heroDB;
+
+USE heroDB;
+
+CREATE TABLE usersLogin(
+  id INT NOT NULL AUTO_INCREMENT,
```

Comment:
shouldn't have to define anything but drop/create database in this file if using sequelize.